parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -357,43 +373,31 @@ retrycmd_curl_file() {
         fi
     done
 }
-systemctl_restart() {
-    retries=$1; wait_sleep=$2; timeout=$3 svcname=$4
+# base systemctl retry command, should not be called directly - use systemctl_restart, systemctl_stop, systemctl_disable
+systemctl_retry_svc_operation() {
+    retries=$1; wait_sleep=$2; timeout=$3 svcname=$4 operation=$5 shouldLogRetryInfo=${6:-false}
```

Comment:
Either change the order of `svcname` and `operation` here or line 381 so that they are aligned?